astropy/io/ascii/tests/test_cds.py

```diff
@@ -0,0 +1,24 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""
+This module tests some methods related to ``CDS`` format
+reader/writer.
+
+Requires `pyyaml <https://pyyaml.org/>`_ to be installed.
+"""
+from io import StringIO
+
+from astropy.table import Table
+
+
+def test_write_data():
+    TABLE = ['S08-229 4625 1.23 1.23 -1.5      ',
```

Comment:
`TABLE` should not be used a variable name here [When in doubt refer to PEP8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names) 😄 .

```suggestion
   test_table_output = ['S08-229 4625 1.23 1.23 -1.5',
```